ddtrace/contrib/aiobotocore/patch.py

```diff
@@ -98,10 +98,9 @@ def _wrapped_api_call(original_func, instance, args, kwargs):
 
         # add args in TRACED_ARGS if exist to the span
         if not aws.is_blacklist(endpoint_name):
-            for name, value in aws.unpacking_args(args, ARGS_NAME, TRACED_ARGS):
-                if name == 'params':
-                    value = {k: truncate_arg_value(v) for k, v in value.items()}
-                span.set_tag(name, (value))
+            operation_args = aws.unpacking_args(args, ARGS_NAME, TRACED_ARGS)
```

Comment:
It still seems like this is duplicated between here and `botocore`, do you think it makes sense to add a single helper `aws.add_span_tags(span, endpoint_name, args)` which checks the `endpoint_name` against the common blacklist, and then unpacks, flattens, and adds necessary tags?